priv/repo/migrations/20180713191518_create_payment_method.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Spender.Repo.Migrations.CreatePaymentMethod do
+  use Ecto.Migration
```

Comment:
Could you please move this `use` below the moduledoc and run `mix format` for this file